apps/src/templates/sectionProgress/VirtualizedDetailView.jsx

```diff
@@ -172,7 +176,12 @@ class VirtualizedDetailView extends Component {
           <span style={styles.bubbleSet}>
             {scriptData.stages[stageIdIndex].levels.map((level, i) => (
               <FontAwesome
-                icon={getIconForLevel(level)}
+                icon={
+                  isLevelAssessment(level) &&
+                  experiments.isEnabled(experiments.MINI_RUBRIC_2019)
+                    ? 'check-circle'
+                    : getIconForLevel(level)
```

Comment:
this logic would be better to put inside `getIconForLevel`. To keep this from having unintended side effects, you might have to pass in an optional parameter here which indicates that this is a situation where you want to include assessment checkmarks.